src/class-fields/zero-zero-initializer-var.case

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2016 the V8 project authors, 2017 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: zero initialized zero var
+info: |
+  1.1 New Productions
```

Comment:
Please remove this, as it won't be relevant when the proposal is merged into the spec